app/web/src/components/Dialog.tsx

```diff
@@ -69,7 +77,25 @@ export function DialogContentText(props: DialogContentTextProps) {
   );
 }
 
+interface DialogTitleProps extends MuiDialogTitleProps {
```

Comment:
```suggestion
interface DialogTitleProps extends Omit<MuiDialogTitleProps, "className"> {
```
So you can use `DialogTitleProps` directly in the component